deploy-git-hook

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+
+if [ -f .git ];
+then
+	LOCATION="$(cat .git | awk '{print $2}')"
+	LOCATION="$LOCATION/hooks/pre-commit"
+else
+	LOCATION=".git/hooks/pre-commit"
+fi
+
+ln -s /home/Jan/Projecten/Github.com/puppet-git-hooks/pre-commit $LOCATION
```

Comment:
This home directory is unique to the user's install.  I suspect "/home/Jan/Porjecten/Github.com/puppet-git-hooks" should be something like "$HOME" or maybe "$HOME/git".  Something that is not user-specific.